R/getStepwiseDivergence.R

```diff
@@ -222,3 +233,36 @@ setMethod("getTimeDivergence",
     return(x)
 
 }
+
+############################ HELPER FUNCTION(S) ################################
+
+.get_mat_from_se <- function (x, assay_name, altexp) {
```

Comment:
Is this code borrowed from some other package?

Reply:
No, I made it myself for the task.

Reply:
Ok looks neat. Just compare to the earlier implemented solutions in mia, and make sure you are not reinventing new solutions (unless they are clearly better and thus justified).

Reply:
I would ofc mention it was adapted from.... as a comment or put it at utils.R header with the other imports from other packages.

Reply:
Which package it was adapted from ?

Reply:
I meant in case if I would have copy pasted the code from another place or package, I would mention as a comment that it was so , or adapted from and mention the source. If it was imported from another package, I will do the import at utils.R for instance of the package.
In this case, I have not copy pasted the code from anywhere, I built based on the steps required for the task.

Reply:
Ok. Just that we always have to take care of open license compatibility issues when reusing code.

Reply:
Hi,
I have just noticed that [mia imports](https://github.com/microbiome/mia/blob/master/R/utils.R#L5) a function from scater [.get_mat_from_sce](https://github.com/Alanocallaghan/scater/blob/master/R/utils_reddim.R#L3).
Shall this be rather used adding to the `getStepwiseDivergence` two more arguments; `dimred` and `n_dimred`?

Reply:
Sounds good!